lib/js/styles/ItemsList.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+	<div>
+		<div v-for="itemsList in itemsLocal" :key="itemsList.title" class="itemDefinition">
+			<div class="itemsList" :class="{ [itemsList.class]: itemsList.class }">
+				<div class="itemsList__row">
+					<h2 class="itemsList__title">{{ itemsList.title }}</h2>
+					<ol class="itemsList__categories">
+						<li
+							v-for="(items, itemCategory) in itemsList.list"
+							:key="itemCategory"
+							class="itemDefinition__category"
+						>
+							{{ itemCategory }}
+						</li>
+					</ol>
+				</div>
+				<div
+					v-for="(items, itemCategory) in itemsList.list"
+					:key="itemCategory"
+					class="itemsList__row"
+				>
+					<h3 :id="'category-' + itemCategory" class="itemsList__category">{{
+						itemCategory
+					}}</h3>
+					<item-color v-if="type === TOKENS_TYPES.COLORS" :items="items" />
+					<item-typography v-if="type === TOKENS_TYPES.TYPOGRAPHY" :items="items" />
+				</div>
+			</div>
+		</div>
+	</div>
+</template>
+
+<style lang="scss" scoped>
+.itemsList {
+	margin: 0 auto;
+	width: 80%;
+
+	&__title {
+		margin-top: 0;
+	}
+
+	&__category {
+		font-weight: var(--typography-font-weight-bold);
+		font-size: var(--typography-font-size-2xl);
+		line-height: var(--typography-line-height-3xl);
+	}
+
+	&__row {
+		border: 1px solid #f3f2f2;
```

Comment:
is it possible to use css token varibale for color?

Reply:
Fixed.